queries/smali/highlights.scm

```diff
@@ -183,9 +177,6 @@
 
 (comment) @comment @spell
 
-(class_definition
-  (comment) @comment.documentation)
-
```

Comment:
So..., what is the alternative, or was it unnecessary from the beginning?

Reply:
Should not have been removed, oops